technologies/job/trifacta/token-auth/jobForm.js

```diff
@@ -0,0 +1,126 @@
+const axios = require('axios');
+const https = require('https');
+const { Response } = require('@saagie/sdk');
+
+const agent = new https.Agent({  
+  rejectUnauthorized: false
+});
+
+const SSL_ISSUES_CODE = 'UNABLE_TO_VERIFY_LEAF_SIGNATURE';
+
+/**
+ * Function to get user flows in Trifacta
+ * @param {Object} entity - Contains entity data including featuresValues.
+ * @param {Object} entity.featuresValues - Contains all the values from the entity features declared in the context.yaml
+ */
+exports.getFlows = async ({ featuresValues }) => {
+  try {
+    const result = await axios.get(
+      `${featuresValues.endpoint.url}/v4/flows`,
+      {
+        httpsAgent: featuresValues.endpoint.ignoreSslIssues && featuresValues.endpoint.ignoreSslIssues.id ? agent : null,
+        headers: {
+          'Authorization': `Bearer ${featuresValues.endpoint.access_token}`
+        }
+      }
+    );
+
+    if (!result) {
+      return Response.empty('No response from Trifacta');
+    }
+
+    const { data: dataResult } = result;
+
+    if (!dataResult) {
+      return Response.empty('No response from Trifacta');
+    }
+
+    const { data: flows } = dataResult;
+
+    if (!flows || !flows.length) {
+      return Response.empty('No datasets availables');
+    }
+
+    return Response.success(
+      flows.map(({ name, id }) => ({
+        id,
+        label: name,
+      })),
+    );
+  } catch (error) {
+    if (error && error.code === SSL_ISSUES_CODE) {
+      return Response.error('Can\'t retrieve flows from Trifacta : SSL error, you can disable SSL issues in Endpoint form', { error: new Error(error.code) });
```

Comment:
```suggestion
      return Response.error('SSL error, you can disable SSL issues in Endpoint form', { error: new Error(error.code) });
```

The prefix is not useful, error message for the user should be concise. `Can\'t retrieve flows from Trifacta` do not add information for the final user.

Reply:
This is a recommendation for all the errors, be concise in the first parameter, for the UI, be explicit in the second one.